spec/models/account_spec.rb

```diff
@@ -21,6 +21,8 @@
   it { should validate_presence_of :display_name }
   it { should validate_presence_of :slug }
   it { should validate_uniqueness_of(:slug).case_insensitive }
+  # TODO: Figure out how to test a model with several unique constraints
+  xit { should validate_uniqueness_of(:tax_id).allow_blank.case_insensitive }
```

Comment:
It seems like model based validation might not be working as well for this. Need to update the assertion or perhaps move the validation to a database constraint to ensure enforcement